### PR TITLE
Merge pull request  Add /about page #9  and Home page #11

### PR DESCRIPTION
**Pull request by Salman-Virji**
About Screen

    URL: /about
    Content: about the bot

### CODE REVIEW

**Comment by Salman-Virji** [comment]
This also completes issue #11 add home screen

File: Views/Home/About.cshtml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+@{
+    ViewData["Title"] = "About";
+}
+
+
+<!-- Refrence for image postioning  -->
+ <!-- https://stackoverflow.com/questions/67081460/how-can-i-align-image-horizontally-in-tailwind-css -->
```
Comment (elhardoum) [inline Views/Home/About.cshtml:7]: This reference isn't needed, you didn't copy any code from SO they only helped you find the right class names.
Just delete lines 6 and 7. Thanks

File: Views/Home/Index.cshtml [hunk 2]
```diff
@@ -2,7 +2,49 @@
     ViewData["Title"] = "Home Page";
 }
 
-<div class="max-w-md m-auto px-3">
-    <h2 class="text-grey-darker uppercase tracking-wide text-center">@ViewData["Title"]</h2>
-    <p class="mt-5 text-grey-darker">Lorem ipsum dolor sit amet, cum te tantas recteque persecuti, sumo appareat torquatos ei sea, alienum insolens cu mei. Oportere scriptorem ea per. Quo ne civibus ancillae, qui doctus iriure ex, eam suas sanctus vivendum an. Dolores offendit assentior at duo, ex brute quaerendum accommodare qui. Inciderint dissentiet vis id, amet offendit ut sed.</p>
-</div>
\ No newline at end of file
+<!-- Refrence for image postioning  -->
```
Comment (elhardoum) [inline Views/Home/Index.cshtml:5]: Same as comment above. Then we're all set to merge the PR

File: Views/Home/Index.cshtml [hunk 3]
```diff
@@ -2,7 +2,49 @@
...
+ <!-- https://stackoverflow.com/questions/67081460/how-can-i-align-image-horizontally-in-tailwind-css -->
```
Comment (elhardoum) [inline Views/Home/Index.cshtml:6]: Same as comment above. Then we're all set to merge the PR

File: Views/Home/About.cshtml (hunk 1 above)
Comment (Salman-Virji) [reply Views/Home/About.cshtml]: Ok comments have been removed

File: Views/Home/Index.cshtml (hunk 3 above)
Comment (Salman-Virji) [reply Views/Home/Index.cshtml]: Ok comments have been removed

**Comment by elhardoum** [comment]
Thanks for your work. Changes are [currently](../actions/runs/1539925191) deploying

**Pull request merged**